src/Client/StoreOnChainWallet.php

```diff
@@ -140,97 +106,76 @@ public function getStoreOnChainWalletTransactions(
 
         $headers = $this->getRequestHeaders();
         $method = 'POST';
 
-        $body = json_encode(
-            [
-                'destinations' => $destinations,
-                'feeRate' => $feeRate,
-                'proceedWithPayjoin' => $proceedWithPayjoin,
-                'proceedWithBroadcast' => $proceedWithBroadcast,
-                'noChange' => $noChange,
-                'rbf' => $rbf,
-                'selectedInputs' => $selectedInputs
-            ],
-            JSON_THROW_ON_ERROR
-        );
+        $body = json_encode(['destinations' => $destinations, 'feeRate' => $feeRate, 'proceedWithPayjoin' => $proceedWithPayjoin, 'proceedWithBroadcast' => $proceedWithBroadcast,
+            'noChange' => $noChange, 'rbf' => $rbf, 'selectedInputs' => $selectedInputs], JSON_THROW_ON_ERROR);
 
         $response = $this->getHttpClient()->request($method, $url, $headers, $body);
 
         if ($response->getStatus() === 200) {
-            return new StoreOnChainWalletTransaction(
-                json_decode($response->getBody(), true, 512, JSON_THROW_ON_ERROR)
-            );
+            return new StoreOnChainWalletTransaction(json_decode($response->getBody(), true, 512, JSON_THROW_ON_ERROR));
         } else {
             throw $this->getExceptionByStatusCode($method, $url, $response);
         }
     }
 
-    public function getStoreOnChainWalletTransaction(
-        string $storeId,
-        string $cryptoCode,
-        string $transactionId
-    ): StoreOnChainWalletTransaction {
-        $url = $this->getApiUrl() . 'stores/' .
-                    urlencode($storeId) . '/payment-methods' . '/OnChain' . '/' .
-                    urlencode($cryptoCode) . '/wallet' . '/transactions' . '/' .
-                    urlencode($transactionId);
+    public function getStoreOnChainWalletTransaction(string $storeId, string $cryptoCode, string $transactionId): StoreOnChainWalletTransaction
+    {
+        $url = $this->getApiUrl() . 'stores/' . urlencode($storeId) . '/payment-methods' . '/OnChain' . '/' . urlencode($cryptoCode) . '/wallet' . '/transactions' . '/' . urlencode($transactionId);
 
         $headers = $this->getRequestHeaders();
         $method = 'GET';
 
         $response = $this->getHttpClient()->request($method, $url, $headers);
 
         if ($response->getStatus() === 200) {
-            return new StoreOnChainWalletTransaction(
-                json_decode($response->getBody(), true, 512, JSON_THROW_ON_ERROR)
-            );
+            return new StoreOnChainWalletTransaction(json_decode($response->getBody(), true, 512, JSON_THROW_ON_ERROR));
+        } else {
+            throw $this->getExceptionByStatusCode($method, $url, $response);
+        }
+    }
+
+    public function patchStoreOnChainWalletTransaction(string $storeId, string $cryptoCode, string $transactionId, string $comment): StoreOnChainWalletTransaction
```

Comment:
Can you please rename it to `updateStoreOnChainWalletTransaction`?. I know it is called "patch" in the API docs but this is likely a mistake. All other patch calls are called "Update" in the API docs.

Reply:
The comment parameter is also `nullable` according to API docs, please make it nullable here too by adding a `?` in front of the type like this `?string $comment`